Formula/subversion.rb

```diff
@@ -57,12 +55,14 @@ class Subversion < Formula
   uses_from_macos "zlib"
 
   on_macos do
+    depends_on "openjdk" => :build if MacOS.version >= :catalina
```

Comment:
Is this purely because we won't have an OpenJDK bottle for those versions? If so, that's a moving target and thus might make sense doing `unless MacOS.version.outdated_release?` instead.

Reply:
```
 ==> brew audit --tap=homebrew/core --except=version
==> FAILED
Full audit --tap=homebrew/core --except=version output
  Error: subversion: undefined method `outdated_release?' for #<Version::NULL>
```
?

Reply:
Hmm, right I guess that's because we're trying to simulate this on a Linux machine in `brew audit` and MacOS version is hardwired to return an object that's not API compatible with `MacOS::Version`:

https://github.com/Homebrew/brew/blob/e1c1157a769963a4222c1a385bcccf8a9f0c736b/Library/Homebrew/os/linux.rb#L43

I guess we could have more hacks like:

https://github.com/Homebrew/brew/blob/e1c1157a769963a4222c1a385bcccf8a9f0c736b/Library/Homebrew/version/null.rb#L45-L53

to cover `outdated_release?`.